src/common/interfaces.ts

```diff
@@ -214,6 +219,7 @@ export interface MergeTaskParameters {
   targetFormat: TileOutputFormat;
   isNewTarget: boolean;
   sources: TaskSources[];
+  taskIndex: IJobResumeState;
```

Comment:
`MergeTaskParameters` should not contains `taskIndex: IJobResumeState;` as its defines the actual merge tasks parameters

Reply:
<img width="484" height="283" alt="image" src="https://github.com/user-attachments/assets/a842da3f-173e-4533-9d1d-08f96d0c1b50" />

Reply:
we dont want `taskIndex` in this scope